adapters/33across/usersync_test.go

```diff
@@ -13,7 +13,7 @@ func Test33AcrossSyncer(t *testing.T) {
 	syncInfo, err := syncer.GetUsersyncInfo("", "")
 	assert.NoError(t, err)
 	assert.Equal(t, "https://ssc-cms.33across.com/ps/?ri=123&ru=localhost%2Fsetuid%3Fbidder%3Dttx%26uid%3D33XUSERID33X", syncInfo.URL)
```

Comment:
Do you want to update the test URL to match the new default?

Reply:
thank you, I had update the test case.